.travis.yml

```diff
@@ -7,16 +7,23 @@ matrix:
   allow_failures:
     - python: 3.6
 install:
-  - pip install flake8 pytest
+  - pip install flake8 pytest psycopg2
   - pip install -r requirements.txt
+services:
+  - postgresql
 before_script:
+  - psql -c 'create database infobase_test;' -U postgres
+  #- psql -c 'create database infogami_test;' -U postgres # required for tests/test_infogami
   # stop the build if there are Python syntax errors
   - flake8 . --count --select=E9,F63,F72 --show-source --statistics
   # exit-zero if there are Python undefined names
   - flake8 . --count --exit-zero --select=F82 --show-source --statistics
   # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
   - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-script: pytest
+script:
+  - pytest infogami
+  - pytest test
+  - pytest tests
```

Comment:
Put this all on one line?  __pytest infogami test tests__

Reply:
yes, good idea! :)